views/editor/nav-options.html

```diff
@@ -119,7 +119,7 @@ <h2 class="snippets-menu-title">{{ gettext("snippetMenuTitle") }}</h2>
                 <div title="{{ gettext("navIncreaseTextSizeIconTitle") }}" id="editor-pane-nav-increase-font" class="font-size-button"></div>
               </div>
             </li>
-            <li>
+            <li id='colortheme-toggle-li'>
```

Comment:
Let's use a different naming convention here that's more about what it is vs. the type, since that could change (also note the use of double vs. single quotes):

```html
<li id="editor-option-colortheme">
...
<li id="editor-option-wraptext">
...
```